### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by CELYNEBIRUNGI**
In this project we: 
1. Added CSS and HTML files
2. Added JavaScript file  and implemented Add and Remove buttons functionalities
3. Configured Linters
4. Added a  Readme file
5. Used localStorage to present user data in the browser

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <h2>${book.author}</h2>
+        <button class="remove-book" data-title="${book.title}">Remove</button>
+      `;
+      bookList.appendChild(bookElement);
+    });
+    const removeButtons = document.querySelectorAll('.remove-book');
+    removeButtons.forEach((button) => {
+      button.addEventListener('click', () => {
+        const title = button.getAttribute('data-title');
+        // eslint-disable-next-line no-use-before-define
```
Comment (Olamarx) [inline index.js]: - [x] I will implore you to remove this linter disabler and try to fix the issue, professionally, you should declare a function before calling it, in fact, that is the essence of the arrow function which came to relegate the function keyword. Kindly make this a part and parcel of you.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let booksCollection = [];
```
Comment (Olamarx) [inline index.js:2]: - [ ] I think you forgot to save your changes in the localStorage, I can see you did a wonderful job but kindly include saving changes in the localStorage as required in the project requirements as attached in the image below.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Add and Remove functionalities ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @CELYNEBIRUNGI , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good preserving data in localStorage.✔️
- [x] Good implementation Add and Remove functionalities  ✔️
- [x] Good usage in correcting GitHub flow.✔️


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**